apps/docs/stories/components/ModularCheckout/introduction.mdx

```diff
@@ -0,0 +1,173 @@
+import { Meta, Source } from '@storybook/blocks';
+import dedent from 'ts-dedent';
+import { extractWebcomponentsVersion, SummaryElement } from '../../utils';
+
+<Meta title="Modular Checkout/Introduction" />
+
+# Justifi Modular Checkout
+
+The Modular Checkout system provides a flexible, composable approach to building checkout experiences. Instead of a monolithic checkout component, you can combine individual components to create custom payment flows that match your specific requirements.
+
+## Architecture Overview
+
+The Modular Checkout system is built around a wrapper component (`justifi-modular-checkout`) that orchestrates validation, tokenization, and checkout completion across multiple child components. This architecture allows you to:
+
+- **Mix and match payment methods**: Include card forms, bank account forms, or saved payment methods as needed
+- **Customize billing collection**: Use full billing forms or simplified postal code collection
+- **Add specialized functionality**: Include components for BNPL (Sezzle), insurance, or other features
+- **Control the user experience**: Arrange components in any layout that suits your design
+
+## Core Components
+
+### Wrapper Component
+
+- **[`justifi-modular-checkout`](/docs/modular-checkout-modular-checkout--docs)**: The orchestrator that manages validation, tokenization, and checkout completion
+
+### Payment Method Components
+
+- **[`justifi-card-form`](/docs/modular-checkout-card-form--docs)**: Secure card payment form with tokenized inputs
+- **[`justifi-bank-account-form`](/docs/modular-checkout-bank-account-form--docs)**: Bank account payment collection
+- **[`justifi-saved-payment-methods`](/docs/modular-checkout-saved-payment-methods--docs)**: Display and select from previously saved payment methods
+- **[`justifi-sezzle-payment-method`](/docs/modular-checkout-sezzle-payment-method--docs)**: Buy Now, Pay Later integration with Sezzle
+
+### Information Collection Components
+
+- **[`justifi-billing-information-form`](/docs/modular-checkout-billing-information-form--docs)**: Complete billing address collection
+- **[`justifi-postal-code-form`](/docs/modular-checkout-postal-code-form--docs)**: Simplified postal code-only billing
+- **[`justifi-checkout-summary`](/docs/modular-checkout-checkout-summary--docs)**: Display checkout totals and descriptions
+
+### Specialized Components
+
+- **[`justifi-season-interruption-insurance`](/docs/modular-checkout-season-interruption-insurance--docs)**: Insurance options integration
+
+## How It Works
+
+### 1. Setup and Configuration
+
+The wrapper component requires basic configuration:
+
+- `auth-token`: Your JustiFi authentication token
+- `account-id`: The account identifier
+- `checkout-id`: The specific checkout session identifier
+- `save-payment-method`: Optional flag to enable payment method saving
```

Comment:
I think this needs a bit more details. I'll need to get back to it tomorrow. Running out of time